services/addons/images/rsu_status_check/README.md

```diff
@@ -0,0 +1,85 @@
+# RSU Status Fetch services
+
+## Table of Contents
+
+- [RSU Status Fetch services](#rsu-status-fetch-services)
+  - [Table of Contents](#table-of-contents)
+  - [About ](#about-)
+  - [Requirements ](#requirements-)
+    - [rsu_ping_fetch](#rsu_ping_fetch)
+    - [rsu_pinger](#rsu_pinger)
+    - [rsu_snmp_fetch](#rsu_snmp_fetch)
+
+## About <a name = "about"></a>
+
+This directory contains three services that run within the CV Manager GKE Cluster as a single microservice called the 'rsu_status_check'.
+
+Both the 'rsu_ping_fetch' and 'rsu_pinger' services populate the CV Manager PostGreSQL database's 'ping' table with the current online statuses of all RSUs recorded in the 'rsus' table. For 'rsu_ping_fetch', these statuses are retrieved directly from a [Zabbix server](https://www.zabbix.com/). For 'rsu_pinger', these statuses are generated by pinging every RSU in the 'rsus' table every 1 minute.
+
+If you have access to a Zabbix server that is tracking RSUs, it is recommended to use the 'rsu_ping_fetch' service. However, the 'rsu_pinger' is an effective alternative.
+
+Another feature the 'rsu_status_check' application provides is a ping data purger that will remove stale ping data from the CV Manager PostgreSQL database to allow for high performance RSU ping queries. The amount of time a message needs to be in the database to be considered stale is configurable with the STALE_PERIOD environment variable. This purger will run once every 24 hours to check for stale ping data in the database.

```

Comment:
Why isn't the purger considered to be its own service? Looks like we're calling it separately in the crontab file.

Reply:
I could allow it to be turned off but it mainly concerns keeping the RSU online status table in PostgreSQL under control. If the ping functions aren't running it won't actually do anything. In the scenario they have a bunch of old ping data remaining it would at least clean it up for them. If tying this to whether thing ping scripts are running or making it separate to control is desirable then I can definitely do that.